.circleci/github-pages.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# This builds docs from current sceptre version and pushes it to the website with lies in separate repository
+set -e
+# show where we are on the machine
+echo "we are in:" $(pwd)
+ls -laF
+# required env vars
+declare -a vars=(
+    CONTAINER_DEST_REPO_DIR # directory in container where dest repo is initialized within container
```

Comment:
`# directory in container where dest repo is initialized within container` -> `# directory in container where dest repo is initialized`.

Reply:
I think a clearer name here would help make the intention of this var clearer. Maybe something like `REPOSITORY_PATH`?